File: Codigo_Fonte_c/Lista 1/lista1_9.c
#include <stdio.h>

int main(){
	float nota1, nota2, media;
	nota1 = 6.3;
	nota2 = 7;
	media = (nota1 + nota2) / 2;
	if (media >= 7){
		printf("Aprovado\n");
	}else{
		printf("Reprovado\n");
	}
	return 0;
}

File: Codigo_Fonte_pascal/lista1/lista1_9.pas
program calcular_media;
var

   nota1, nota2, media : real;

begin
     write('Informe a primeira nota: ');
     readln(nota1);
     write('Informe a segunda nota: ');
     readln(nota2);

     media := (nota1 + nota2) / 2;
     //writeln(media);

     writeln('Situação do Aluno:');
     if (media >= 7) then
          writeln('Aprovado')
     else
          writeln('Reprovado');

     readln;
end.
